win32ss/user/winsrv/concfg/font.c

```diff
@@ -70,9 +74,10 @@ CreateConsoleFontEx(
 
     if (!IsValidConsoleFont(FaceName, CodePage))
     {
-        StringCchCopyW(FaceName, LF_FACESIZE, L"Terminal");
```

Comment:
So now, what font is it going to use if `IsValidConsoleFont` fails? The `lf.lfFaceName` below would be initialized with the name of that invalid font.

Reply:
```txt
    if (!IsValidConsoleFont(FaceName, CodePage))
    {
        if (IsCJKCodePage(CodePage))
        {
            MessageBoxW(NULL, FaceName, NULL, 0); // Added
            if (pbCJKFontFallback)
                *pbCJKFontFallback = TRUE;
            lf.lfCharSet = ANSI_CHARSET;
        }
    }
```

Reply:
Let's please stop piling hacks in top of my code, plz...